OneCommunityGlobal/HighestGoodNetworkApp
# Fix WBS task Responsiveness

# Description
Fix WBS Tasks Responsiveness. WBS tasks are not responsive on mobile and it only shows the first 5 columns. 

## Mainly changes explained:
I have displayed all columns on mobile. You have to scroll to the right to see.
I have made all toggle/ filter buttons look better on mobile by giving them display flex.
…

## How to test:
check into the current branch
do `npm install` and `...` to run this PR locally
log as admin user
go to dashboard→ Other links → Projects → AAA USE THIS PROJECT  → Jae's WBS - USE THIS 

## Screenshots or videos of changes:
Before 
![Screenshot 2023-03-20 083659](https://user-images.githubusercontent.com/82662768/226238433-7cefaea1-cda3-4d53-8f8b-7f36c8492a1c.jpg)

after
https://user-images.githubusercontent.com/82662768/226238408-2133366b-dcb9-4282-a75f-adeb277cf290.mp4


## Note:
I have also tried a second option without a scrollbar but that didn't work well.
